api/src/main/java/org/pmiops/workbench/exfiltration/EgressLogService.java

```diff
@@ -151,36 +158,44 @@ private AuditEgressRuntimeLogEntry toRuntimeLogEntry(FieldValueList fvl) {
   }
 
   private String getDatasetId(Optional<SumologicEgressEvent> sumologicEgressEvent) {
-    // If getGceEgressMib is more than zero, we see this egress triggered by GCE, otherwise, it's
-    // Dataproc.
-    // TODO(yonghao): Revisit this logic and probably make an enum once we understand what does GKE
-    // sumologic event looks like.
-    boolean isGce =
-        sumologicEgressEvent
-            .map(SumologicEgressEvent::getGceEgressMib)
-            .map(d -> d > 0)
-            .orElse(false);
-    return isGce
+    return isGce(sumologicEgressEvent)
```

Comment:
nit: this can now be: 
```
return workbenchConfigProvider.get().firecloud.workspaceLogsProject + "." + getTextColumnName(sumologicEgressEvent);
```
or the String.format() equivalent